docs/app/pages/components/sections/panels/item-display-panel-inline/item-display-panel-inline.component.html

```diff
@@ -3,7 +3,7 @@
 <div class="demo-attributes item-display-panel-inline-container">
     
     <div class="table-responsive preview-pane-table-demo">
-        <table class="table table-hover item-display-panel-inline" [class.animate]="animate" [class.item-display-panel-visible]="visible">
+        <table class="table table-hover item-display-panel-inline animate" [class.item-display-panel-visible]="visible">
             <thead>
```

Comment:
In IE we are also getting some odd layout:

![image](https://user-images.githubusercontent.com/20795331/28066162-ca43c644-6633-11e7-81e7-ec21f12eb870.png)

![image](https://user-images.githubusercontent.com/20795331/28066172-d16ec306-6633-11e7-9c7b-97216e679679.png)